wiki/Community/Forum/Off-Topic/en.md

```diff
@@ -7,3 +7,74 @@ stub: true
 <!-- TODO: A LOT could be written here: https://github.com/ppy/osu-wiki/issues/2804 -->
 
 [**Off-Topic**](https://osu.ppy.sh/community/forums/52) is a forum meant for the community to have conversations without any particular topic in focus. Threads can often be random, change topic, and/or have multiple topics simultaneously. Topics can range anything from daily life, opinions about the game, role play, jokes and meta content, nonsense, or anything in between.
+
+## History
+
+The "General Discussion" subforum initially hosted any gameplay unrelated topic the people wanted to talk about. As forum games started polluting the subforum, people suggested making a separate subforum for "self perpetuating threads". On 19 February 2009 ppy created the suggested subforum named "Forum Games/Random".
+
+The new subforum created a distinction that "General Discussion" is to be used for serious discussions while "Forum Games/Random" is to be used for forum games and non-serious discussions. Given the nature "Forum Games/Random" subforum, people would often interrupt games hosted with random conversations. On 2 February 2010 "Forum Games/Random" was renamed to "Off-Topic" and the "Forum Games" subforum was created.
+
+Two more subforum would later be created: "Mafia" subforum on 14 September 2011, and "Surveys" subforum on 5 March 2014.
+
+## Traditions and Culture
+#### Valhalla
```

Comment:
headings [must not skip a level](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#sections), so every `####` needs to be trimmed to `###`